package work;

public class Ex02 {

	public static void main(String[] args) {
		
		System.out.println("try - catch - finally 블록에 대한 설명 중 틀린 것은 무엇입니까?");
		
		System.out.println("1)try{}블록에는 예외가 발생할 수 있는 코드를 작성한다.");
		System.out.println("2)catch{}블록은 예외의 종류별로 여러 개를 작성할 수 있다.");
		System.out.println("3)try()블록에서 return문을 사용하면 finally()블록은 실행되지 않는다.   -  x");
		System.out.println("4)catch()블록은 예외의 종류별로 여러 개를 작성할 수 있다.");

	}

}
